### DISCUSSION TITLE
Result is not getting formatted for display

### DISCUSSION THREAD

**Discussion by jwhoami** [rest-nvim/rest.nvim#253] [Q&A]
Hi,

I am using rest.nvim with lazyvim on ubuntu 22.04. The request is sent and and results gets displayed fine.
However the result is not formatted, especially the content between #+RESPONSE and #+END

I installed jq and tidy on ubuntu and also installed the jq formatter using Mason on Lazyvim,

What am i missing.

Thanks

![i1](https://github.com/rest-nvim/rest.nvim/assets/74318608/60055f98-dc85-4186-b942-c0fcfd582d91)

My configuration is as follows

`return {

  "rest-nvim/rest.nvim",

  dependencies = { { "nvim-lua/plenary.nvim" } },

  ft = "http",

  config = function()

    require("rest-nvim").setup({

      result_split_horizontal = true,

      result_split_in_place = false,

      skip_ssl_verification = false,

      encode_url = true,

      highlight = {
        enabled = true,
        timeout = 150,
      },

      result = {

        show_url = true,

        show_curl_command = true,

        show_http_info = true,

        show_headers = true,

        formatters = {
          json = "jq",
          html = function(body)
            return vim.fn.system({ "tidy", "-i", "-q", "-" }, body)
          end,

        },

      },

      jump_to_request = false,

      env_file = ".env",

      custom_dynamic_variables = {},

      yank_dry_run = true,

    })

  end,

  keys = {

    { "<leader>rr", "<Plug>RestNvim<cr>", desc = "Send request" },

  },

}`

**Comment by jwhoami** [comment]
This a hack that I am using just in case someone needs it

Set the buffer as modifiable
`: set ma`

Select line #16 (in the image above) in visual mode and run 
`:!jq`

And the line get formatted.

**Reply by knownasnaffy** [reply]
Thanks, mate! I've been trying to figure this out for an hour now.

Created a macro for it and it's working just fine now.

**Comment by NTBBloodbath** [comment]
Hey, the issue has been fixed by #255 so I'm closing this discussion as outdated :)
